authors/settings.py

```diff
@@ -138,6 +142,6 @@
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
+       'rest_framework_jwt.authentication.JSONWebTokenAuthentication',
```

Comment:
@koechkevin I'm not so familiar with both of them, can you help me understand the reason behind upgrading from manual jwt to djangorestframework-jwt?

Reply:
We started the project with bugs. While I was debugging, I made the change to enable testing of the application.